SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -815,6 +813,14 @@ void CDriver::Geometrical_Preprocessing_FVM(CConfig *config, CGeometry **&geomet
   if (rank == MASTER_NODE) cout << "Checking for periodicity." << endl;
   geometry[MESH_0]->Check_Periodicity(config);
   
+  /*--- Compute mesh quality statistics on the fine grid. ---*/
+  
+  if (!fea) {
```

Comment:
Thanks, I agree we need a larger sweep to see what can be trimmed.

Reply:
Maybe it would make sense to create a child class of CPhysicalGeometry or CGeometry for the FEA mesh (not now but in the future).

Reply:
Absolutely. Let's come back to that when we split up PhysicalGeometry. I have some ideas about splitting that one.. it could easily be 3 or more separate classes.